types/knockout/index.d.ts

```diff
@@ -4,21 +4,23 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-interface KnockoutSubscribableFunctions<T> {
+interface KnockoutExtensionFunctions {
     [key: string]: KnockoutBindingHandler | undefined;
+}
 
-	notifySubscribers(valueToWrite?: T, event?: string): void;
+interface KnockoutSubscribableFunctions<T> {
+    notifySubscribers(valueToWrite?: T, event?: string): void;
 }
+type KnockoutSubscribableExtensionFunctions<T> = KnockoutSubscribableFunctions<T> & KnockoutExtensionFunctions;
```

Comment:
why not just use `extends` for interfaces:
```ts
interface KnockoutSubscribableExtensionFunctions<T> extends KnockoutSubscribableFunctions<T>, KnockoutExtensionFunctions {
}
```
it is usually easier for TS deal with interfaces rather than intersections.
Same with `KnockoutComputedExtensionFunctions`